vendor/assets/javascripts/jquery.jeditable.js

```diff
@@ -154,20 +154,16 @@
                 
                 /* figure out how wide and tall we are, saved width and height */
                 /* are workaround for http://dev.jquery.com/ticket/2190 */
-                if (0 == $(self).width()) {
```

Comment:
This workaround overrides the input `width` and `height` values, and causes the issue after an edit.